es6-modules/src/components/todo-form.js

```diff
@@ -0,0 +1,45 @@
+import createElement from '../lib/create-element';
```

Comment:
This should be imported from `lib/index.js` with `import { createElement } from '../lib';`